static/docs/commands-reference/checkout.md

```diff
@@ -43,15 +43,15 @@ files that are under DVC control. For convenience a Git hook is available,
 simply by running `dvc install`, that will automate running `dvc checkout` after
 `git checkout`. See `dvc install` for more information.
 
-Note, this command does NOT copy any files (except when the `cache.type` config
-option is `copy`). Instead, DVC uses links to perform data file restoration.
-This is crucial for large files where checking out a 50Gb file might take a few
-minutes. With DVC links, restoring a 50Gb data file will take less than a
-second.
-
-> Note that when linking files takes longer than expected and `cache.type` is
```

Comment:
could we change a language - "expected" is not defined probably. Either we can specify the default value we use (we need to confirm with @pared ) or just change the terminology

Reply:
It's 10 seconds. See https://github.com/iterative/dvc/blob/ce83231128d77f3c2c8f8c55f9baf77db0e5493a/dvc/remote/local/slow_link_detection.py#L11

Reply:
Updated, please check.